pressbooks/pressbooks-multi-institution
# fix: vite dev dependency deprecation

### Problem
* We use https://github.com/kucrut/vite-for-wp to build assets and CSS, the latest version contained some deprecations and new functions

### Solution
* Bump to the latest version and fix our entry point file

### How to test
* Checkout the branch and work on js files that you want to work with and review if the build process works as expected, it should work with `npm run dev` and `npm run build`
